### PR TITLE
Add lint and fix lint issues

### PR DESCRIPTION
**Pull request by KiranVarma02**
### Change description ###

Add lint and fix lint issues

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ *] No
```

### CODE REVIEW

**Review by akomisarek — DISMISSED** [review]
Do we run lint on pre-commit, do you want to run it?

**Comment by KiranVarma02** [comment]
> Do we run lint on pre-commit, do you want to run it?

Shall we do it just on push? Don't mind. We can do --no-verify if we choose not to when its enabled.

File: src/test/a11y/a11y.ts [hunk 1]
```diff
@@ -21,13 +21,26 @@ class PallyIssue {
   typeCode: number
 }
 
-describe('Accessibility', () => {
+function ensurePageCallWillSucceed (url: string): Promise<void> {
+  return agent.get(url)
```
Comment (mounikahmcts) [inline src/test/a11y/a11y.ts:25]: Do we need to handle post calls as well? Not sure if we use them in our application. 
pa11y('http://example.com/', {
    headers: {
        'Content-Type': 'application/json'
    },
    method: 'POST',
    postData: '{"foo": "bar", "bar": "baz"}'
});

Comment (Jamesnelson117) [reply src/test/a11y/a11y.ts:25]: @mounikahmcts Interesting. I'm new to a11y so correct me if i'm wrong but a11y is just to test the accessibility of a view no? What would a11y check in a post?

File: README.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ Running the linting:
 $ yarn lint
 ```
 
+Running the linting with auto fix:
+```bash
+$ yarn lint --fix
+```
+
```
Comment (Jamesnelson117) [inline README.md:73]: Hmm, is this needed in the readme?

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
     "cichecks": "yarn && yarn sass && yarn lint  && yarn test && yarn test:routes &&  yarn test:a11y",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx && sass-lint -v -q",
+    "lint": "sass-lint -v -q && eslint . --ext .js,.jsx,.ts,.tsx ",
```
Comment (Jamesnelson117) [inline package.json:11]: why the change?

Comment (KiranVarma02) [reply package.json:11]: yarn lint -- fix doesn't work otherwise

File: README.md (hunk 2 above)
Comment (KiranVarma02) [reply README.md:73]: just info :)

File: src/test/a11y/a11y.ts (hunk 1 above)
Comment (KiranVarma02) [reply src/test/a11y/a11y.ts:25]: I think a11y is just accessibility but we will test post calls in our route tests.

Comment (KiranVarma02) [reply src/test/a11y/a11y.ts:25]: just to add, i just fixed lint issues in this file as part of this PR. I didn't do anything else :).

Comment (mounikahmcts) [reply src/test/a11y/a11y.ts:25]: @Jamesnelson117  @KiranVarma02  I meant testing a page which can be rendered only via post call. But yeah, we can cover them in route tests if the scenario comes

**Review by Jamesnelson117 — APPROVED** [review]
LGTM
